packages/protocol/contracts/Staking.sol

```diff
@@ -269,6 +273,8 @@ contract Staking is ERC900, IStakingLocking, IsContract, TimeHelpers {
      * @param _user Owner of lock
      * @param _newLockManager New lock manager
      */
+    // Note: this is more like a transfer than "set"
+    // Note: is this function ever used? Do we have a use case for it?
     function setLockManager(address _user, address _newLockManager) external {
```

Comment:
I'm leaning on removing this... this introduces some more surface area and would have to be built into the lock manager itself to be utilized (which by itself could be kind of scary in terms of arbitrarily moving a lock).

Reply:
Sounds good to me. Pinging @facuspagnuolo to make sure he is aware too.